docs/resources/kms_key.md

```diff
@@ -62,6 +69,8 @@ In addition to all arguments above, the following attributes are exported:
 * `expiration_time` - Expiration time.
 * `creation_date` - Creation time (time stamp) of a key.
 * `rotation_number` - The total number of key rotations.
+* `origin` - The source of the kms key.
+* `key_usage` - The key usage.
```

Comment:
These fields already exist in Argument and do not need to be specified repeatedly in Attribute.